doc/classes/InstancePlaceholder.xml

```diff
@@ -18,13 +18,14 @@
 			<argument index="1" name="custom_scene" type="PackedScene" default="null">
 			</argument>
 			<description>
+				Not thread-safe. Use [method Object.call_deferred] if calling from a thread.
```

Comment:
It would be good to add a description of what the method actually does too. We currently rely on descriptions being empty to know what needs work, and this addition will make this appear as documented even though it doesn't say much about its use.

Reply:
Nevermind, this is done by #35530 which just needs a review/merge. It can be rebased to include this note.